bin/tessel-install-drivers.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var logs = require('../lib/logs');
+var child_process = require('child_process');
+
```

Comment:
@rwaldron the first item within `.install` is an OS check. Are you opposed to the way that I did it? (which was copied from the T1 CLI).